en/snap/with-plugins.md

```diff
@@ -420,226 +471,272 @@ Customer receives email notification| Receive notification|  Check MAP menu sett
 </details>
 <hr><br><br>
 
-### Opencart
-<hr>
-
-Midtrans ❤️ Opencart! This is official Midtrans extension for the OpenCart E-Commerece platform.
+## OpenCart
+Midtrans ❤️ OpenCart! This is official Midtrans extension for the OpenCart E-commerce platform.
 
-#### Requirements:
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+* OpenCart minimal 2.0 or later **|** Tested up to v3.0
+* PHP version 5.4 or later
+* MySQL version 5.0 or later
+* Midtrans plugin for OpenCart: For OpenCart v3.0 [[GitHub](https://github.com/Midtrans/Midtrans-Opencart3) | [Zip](https://github.com/Midtrans/Midtrans-Opencart3/archive/master.zip) ]; For OpenCart v2.3 [ [GitHub](https://github.com/Midtrans/SNAP-Opencart-2.3/) | [Zip](https://github.com/Midtrans/SNAP-Opencart-2.3/archive/master.zip)]; For OpenCart v2.o, OpenCart v2.1, or OpenCart v2.2 [ [GitHub](https://github.com/veritrans/SNAP-Opencart) | [Zip](https://github.com/veritrans/SNAP-Opencart/archive/master.zip)]
```

Comment:
<img width="788" alt="Screen Shot 2020-12-22 at 12 09 24" src="https://user-images.githubusercontent.com/13027142/102851536-a4536a00-444e-11eb-9363-c5a4f9b4049e.png">
That is a long and ugly line. The visual of the original (with sub-bullet-point) is better:

<img width="458" alt="Screen Shot 2020-12-22 at 12 09 13" src="https://user-images.githubusercontent.com/13027142/102851604-e11f6100-444e-11eb-80c8-a7be945dcfdd.png">

please fix

Also, `v2.o`??? careful please, it is`v2.0`